atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/Value.java

```diff
@@ -82,6 +82,11 @@ private Value(byte[] contents, long timestamp) {
 
     public static final Function<Value, byte[]> GET_VALUE = Value::getContents;
 
+    public long sizeInBytes() {
+        // one byte added for the timestamp
```

Comment:
comment nit: the timestamp is 8 bytes for single long

```suggestion
        // one long added for the timestamp
```